### ISSUE TITLE
[Accessibility][RTE Plugin] Lack of aria-label attribute on links

### ISSUE THREAD

**Issue by Gajalakshmi0408** [adobe/aem-core-wcm-components#2070]
**Current behaviour**
The OOTB RTE Plugin do not have Accessibility tab/aria-label attribute and <strong></strong> tag.

**Expected behaviour:**
To introduce as part of RTE plugin

- aria-label attribute/Accessibility tab : to make the content inside it to be accessibility compliant
- **strong** tag along with bold/italic to meet the HTML standards.
- **span** tag option for RTE

**Comment by Gajalakshmi0408** [comment]
@gabrielwalt 

1. If we are using any contact number/numeric values in the RTE plugin, The AT users will not be able to get it right.
It would be nice if we can have a aria-label attribute that describes the purpose of it. 

In the below screenshot, if we can have aria-label attribute, it would help.

<img width="414" alt="RTE" src="https://user-images.githubusercontent.com/76493052/162184056-d82d06f6-e5b8-4255-824d-c86fa51eee66.PNG">
2. Currently we have Bold & Italic tags available as part of RTE Plugin. Since **bold** tag is purely styling tag which is ignored by many accessibility tools, it is good to have **strong** tag also.
<img width="396" alt="RTE1" src="https://user-images.githubusercontent.com/76493052/162184855-22743b52-1fa7-4fc9-b93f-6dcdcff744de.PNG">
3. If we add any content inside the RTE, by default paragraph tag is appended to the content. Currently we have paragraph & heading tags. We need to have span tag also to choose from the available options.

<img width="153" alt="RTE2" src="https://user-images.githubusercontent.com/76493052/162185900-17f75621-f28f-4a54-ac5f-ffb783c40578.PNG">

**Comment by Gajalakshmi0408** [comment]
@gabrielwalt If the requested options are queued as part of future release, can i be notified in the same ticket?

**Comment by gabrielwalt** [comment]
Hi @Gajalakshmi0408, that's a great point, and I'd support it. The RTE however is a widget the Core Components are only using and that is not technically part of this GitHub repository. Could you therefore open a corresponding Adobe support ticket?

I'm closing this issue for the time being as this cannot be fixed meaningfully within the scope of this GitHub repository.
